spectral.yaml

```diff
@@ -15,7 +15,7 @@ rules:
   pascal-case-name-rule:
     severity: error
     recommended: true
-    message: OperationId should be PascalCased.
+    message: OperationId should be pascalCased.
```

Comment:
Nit: I think we kept P as Uppercase to annotate that first letter should be Uppercase int OperationId